test/Elastica/Aggregation/GeoBoundsTest.php

```diff
@@ -8,7 +8,7 @@
 
 class GeoBoundsTest extends BaseAggregationTest
 {
-    protected function _getIndexForTest()
+    private function getIndexForTest()
```

Comment:
Welcome to Elastica ;-) Seems like we should make this switch across the product then. Perhaps we can split it up in 2 parts. First migrate all tests and the library. On the library side it will be obviously a BC break so we should do it as soon as we decide that we are potentially not going to do any further 5.x releases. ES 6.0.0-alpha1 should be out in a few weeks. That sounds like a good timing from my point of view.